packages/grafana-data/src/transformations/transformers/rowNumberToField.ts

```diff
@@ -11,7 +11,7 @@ export interface RowNumberToFieldTransformerOptions {}
 export const rowNumberToFieldTransformer: DataTransformerInfo<RowNumberToFieldTransformerOptions> = {
   id: DataTransformerID.rowNumberToField,
   name: 'Row number to field',
-  description: 'Add the row number of the data frame as a field',
+  description: 'Extract row number to column',
```

Comment:
@rdubrock was this a commit derp? Looks unrelated to the rest of this commit.

Reply:
Yarp, it was derp